packages/nextjs/components/example-ui/ContractInteraction.tsx

```diff
@@ -77,7 +79,7 @@ export const ContractInteraction = () => {
 
           <div className="mt-4 flex gap-2 items-start">
             <span className="text-sm leading-tight">Price:</span>
-            <div className="badge badge-warning">0.01 ETH + Gas</div>
+            <div className="badge badge-warning">0.01 {configuredNetwork.nativeCurrency.symbol} + Gas</div>
```

Comment:
This is just an example, inside example-ui. So no need to make it dynamic. You can remove it.

Reply:
Ya but removing it from the example could be confusing right if they switch chains.
Why not keep the example to the point as well 🤷‍♂️

Reply:
It won't be confusing, having it dynamic might be more confusing. 

Maybe you are missing some context of how/why ExampleUI was created in the first place.

Reply:
Ya that could be the case, can you please share any references?